testing/resources/test_repo/imports_module/__manifest__.py

```diff
@@ -0,0 +1,10 @@
+{
+    'name': 'Test Import Messages',
```

Comment:
Are you sure you need a new module?

CAn't you reuse some of the already existing broken modules?

Reply:
I think it is cleaner to have test files for each message, or at least have them groupd. pylint does some variation of this and I find it easier to work with, since you don't have to look around different broken modules to find the offending code.

Reply:
I think differently, creating a new module will trigger errors in other checkers. Actually, you had to modify unrelated expected errors for this to work.